app/viewmodels/checkAnswers/AssessmentsSummary.scala

```diff
@@ -17,45 +17,41 @@
 package viewmodels.checkAnswers
 
 import controllers.routes
-import models.AssessmentAnswer.NoExemption
 import models.ott.CategoryAssessment
 import models.{CheckMode, UserAnswers}
 import pages.AssessmentPage
 import play.api.i18n.Messages
 import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
 import viewmodels.govuk.summarylist._
 import viewmodels.implicits._
-
+import viewmodels.AssessmentCyaKey
 object AssessmentsSummary {
 
   def row(
     recordId: String,
     answers: UserAnswers,
     assessment: CategoryAssessment,
-    indexOfThisAssessment: Int,
-    numberOfAssessments: Int
+    indexOfThisAssessment: Int
   )(implicit messages: Messages): Option[SummaryListRow] =
-    answers.get(AssessmentPage(recordId, indexOfThisAssessment)).flatMap { answer =>
-      val descriptiveText = if (answer == NoExemption) {
-        Some("assessment.exemption.none.checkYourAnswersLabel")
-      } else {
-        assessment.exemptions
-          .find(x => x.id == answer.toString)
-          .map(x => messages("assessment.exemption", x.code, x.description))
-      }
+    answers.get(AssessmentPage(recordId, indexOfThisAssessment)).map { answer =>
+      val listItems = assessment.exemptions
```

Comment:
been asked in the PRA to not use x as variable names, can we change it to something more descriptive? :D

Reply:
Sure! Did away with the variable entirely.